components/raftstore-v2/src/operation/query/local.rs

```diff
@@ -552,7 +564,7 @@ mod tests {
         let leader2 = prs[0].clone();
         region1.set_region_epoch(epoch13.clone());
         let term6 = 6;
-        let mut lease = Lease::new(Duration::seconds(1), Duration::milliseconds(250));
+        let mut lease = Lease::new(Duration::seconds(10), Duration::milliseconds(250));
```

Comment:
It would look more reasonable that `renew_leader_lease_advance_duration` is changed to 2.5s according to the new max lease?

Reply:
Good suggestio.